dags/synapse-dataset-to-croissant.py

```diff
@@ -567,8 +778,14 @@ def query_snowflake_and_push_croissant_file(root_carrier_context: Dict, dataset_
     datasets = query_synapse_dataset_collection_for_datasets.partial(root_carrier_context=root_carrier_context).expand_kwargs(
         get_dataset_collections(root_carrier_context=root_carrier_context))
 
+    combined_dataset_lists = combine_dataset_lists(
```

Comment:
Should we just upload a synapse table with the mapping:

Table name: croissant-files
schema:
dataset_id, croissant_link?

Reply:
Yeah. I was intending to knock this out. Use the existing service account to push out data to a synapse table.

Schema would need to be:
Dataset syn id
Dataset Version
Dataset collection syn id
Croissant file s3 link